docs/documentacao/arquitetura.md

```diff
@@ -0,0 +1,148 @@
+#   Arquitetura do projeto
+
+### Finalidade
+
+<p align = "justify">O documento tem como objetivo estabelecer uma visão da arquitetura de software adotada no sistema Visualeasy. Com isso ele tem como finalidade evidenciar com clareza as decisões arquiteturais que foram tomadas no projeto. Disponibilizando as informações essenciais para as pessoas desenvolvedoras e demais envolvidos a respeito da aplicação e das tecnologias que serão utilizadas no desenvolvimento do projeto.</p>
+
+### Escopo
+<p align = "justify"> Essa documentação foi elaborada sobre a visão de arquitetura de software que será utilizada para a implementação do projeto Visualeasy, de forma a evidenciar as decisões tomadas. Serão abordados os padrões de arquitetura, os frameworks que serão usados no desenvolvimento do projeto. O objetivo é fornecer visualização gráfica de séries temporais, proporcionando uma melhor visualização dos dados fornecidos à aplicação e permitindo uma tomada de decisão com base na análise dos dados.</p>
```

Comment:
Mencionar as questões de qualidade que serão levadas em consideração durante o desenvolvimento; Colocar "framework" em itálico.